chips/README.md

```diff
@@ -0,0 +1,333 @@
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.decathlon.vitamin.chips.VitaminFilterMediumChip
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Filter medium" />
+
+    <com.decathlon.vitamin.chips.VitaminFilterSmallChip
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Filter small" />
+
+</com.google.android.material.chip.ChipGroup>
+```
+
+or
+
+```xml
+<com.google.android.material.chip.ChipGroup
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.chip.Chip
+        style="?attr/chipFilterMedium"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Filter medium" />
+
+    <com.google.android.material.chip.Chip
+        style="?attr/chipFilterSmall"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Filter small" />
+
+</com.google.android.material.chip.ChipGroup>
+```
+
+### Input
+
+```xml
+<com.google.android.material.chip.ChipGroup
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.decathlon.vitamin.chips.VitaminInputMediumChip
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Input medium" />
+
+    <com.decathlon.vitamin.chips.VitaminInputSmallChip
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Input small" />
+
+</com.google.android.material.chip.ChipGroup>
+```
+
+or
+
+```xml
+<com.google.android.material.chip.ChipGroup
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.chip.Chip
+        style="?attr/chipInputMedium"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Input medium" />
+
+    <com.google.android.material.chip.Chip
+        style="?attr/chipInputSmall"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Input small" />
+
+</com.google.android.material.chip.ChipGroup>
+```
+
+You can also use icons:
+
+```xml
+<com.google.android.material.chip.ChipGroup
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.decathlon.vitamin.chips.VitaminInputMediumChip
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:chipIcon="@drawable/ic_vtmn_map_pin_line"
+        android:text="Input medium with icon" />
+
+</com.google.android.material.chip.ChipGroup>
+```
+
+If you want to use image (like profile picture), you have a specific type for that (to avoid blue tint). You have to round the image by yourself.
```

Comment:
That could be great to show how to get a bitmap from Glide, convert it to a drawable and initialize a chip with this drawable.

Reply:
Good idea, I will try that and put the expected code here